d2bs/kolbot/sdk/uiflag.txt

```diff
@@ -1,37 +1,37 @@
-0x01	Inventory open
-0x02	Character Stat screen open
-0x03	quick skill
-0x04	skill
-0x05	Chat box open (typing something)
-0x06
-0x07
-0x08	npc menu
-0x09	Esc menu?
-0x0A	Automap is on
-0x0B	config controls
-0x0C	Shop open at NPC
-0x0D	alt show items
-0x0E	cash
-0x0F	quest
-0x10
-0x11	questlog button
+0x01    Inventory screen open
+0x02	Character screen open
```

Comment:
0x02 the original statement Character Stat screen is clearer than Character Screen